lib/facebook_ads.rb

```diff
@@ -45,9 +45,9 @@ def configure
 
   require 'facebook_ads/ruby2patch'
 
+  require 'facebook_ads/session'
```

Comment:
because it does require faraday which is now needed in config for the default value